Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.8-slim as base
+
+# Build python modules first
+FROM base as builder
+
+RUN apt-get update \
+    && apt-get install gcc -y \
+    && apt-get clean
+
+RUN python -m venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+COPY ./opni-insights-service/requirements.txt /requirements.txt
+
+RUN pip install -r /requirements.txt
+
+# Create the final image
+FROM base
+EXPOSE 80
+
+COPY --from=builder /opt/venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+COPY ./opni-insights-service/app /app
+
+WORKDIR "app"
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "80"]
```

Comment:
We should be using the `rancher/opni-python-base:3.8` image to share layers with the other images.  If required we can add more prerequisites to that base image.

Reply:
Okay, updating that now. Is kubernetes v12.0.1 a part of this Docker image currently?